src/cleanvision/imagelab.py

```diff
@@ -87,15 +148,54 @@ def _get_issues_to_compute(
             }
         return to_compute_issues_with_params
 
-    def find_issues(self, issue_types: Optional[Dict[str, Any]] = None) -> None:
+    def find_issues(
+        self, issue_types: Optional[Dict[str, Dict[str, Any]]] = None
+    ) -> None:
+        """Finds issues in the dataset.
+        If issue_types is empty or not given, dataset is checked for all default issue types.
+
+        Parameters
+        ----------
+        issue_types : Dict[str, Any], optional
+            Dict with issue_types to check as keys
+            The value of this dict is a dict containing hyperparameters for each issue type
+
+        Examples
+        --------
+        To check for all default issue types use
+
+        .. code-block:: python
+            imagelab.find_issues()
+
+        To check for specific issue types with default settings
+
+        .. code-block:: python
+            issue_types = {
+                "dark": {},
+                "blurry": {}
+            }
+            imagelab.find_issues(issue_types)
+
+        To check for issue types with different hyperparameters.
+
+        .. code-block:: python
+            issue_types = {
+                "dark": {"threshold": 0.1},
+                "blurry": {}
+            }
+            imagelab.find_issues(issue_types)
+
+        Different issue types can have different hyperparameters
+
+        """
         to_compute_issues_with_params = self._get_issues_to_compute(issue_types)
```

Comment:
If `issue_types` is empty list, why do we check for default issue types? I think we should only check for default issue types when this is None. When it is empty list, should either raise ValueError or print("No issue types specified to detect") or something like that.

@elisno @sanjanag thoughts?

Reply:
Checking for default issues only when None, sounds the right behaviour to me for `find_issues`

Reply:
This was already the case in Datalab in one of the internal methods that sets necessary arguments based on those conditions.

I've added an early return for this comment, but that's not covered by tests.